model/src/test/resources/oracle/kubernetes/weblogic/domain/v2/domain-sample-3.yaml

```diff
@@ -29,6 +31,9 @@ spec:
         annotations:
           time: midnight
     serverPod:
+      env:
```

Comment:
It seems that we have two ways of specifying Java Options.

  serverPod:
    javaOptions: "-add-this"

and

    serverPod:
      env:
      - name: JAVA_OPTIONS
         value: "-server"

Are both configurable at all (domain/server/cluster) levels? How do they work together if specified in both javaOptions and envs?

Reply:
It is confusing to have two ways of doing the same thing. We don't have to be backward compatible in 2.0.

Reply:
Russ, I thought we had agreed that we would leave the javaOptions field out of the domain schema and just use env/JAVA_OPTIONS?  It's simpler just to have one schema element.